apps/api/tasks.py

```diff
@@ -49,14 +47,14 @@ def setup_connect_channels_for_bots(self, connect_id: UUID, experiment_data_map:
     # https://github.com/dimagi/open-chat-studio/issues/1046
```

Comment:
Remove TODO

Reply:
Removed it in [this coming PR](https://github.com/dimagi/open-chat-studio/pull/1071/commits/78d5f4a860afe3cebf27637168321dc87b22ba68)